django/contrib/admin/templates/admin/change_form.html

```diff
@@ -73,15 +73,39 @@
             $(document).ready(function() {
                 $('.add-another').click(function(e) {
                     e.preventDefault();
-                    showAddAnotherPopup(this);
+                    var event = $.Event('django:add-another-related');
+                    $(this).trigger(event);
+                    if (!event.isDefaultPrevented())
```

Comment:
I think it's less error prone to always include the brackets even on single statement if statements.